src-docs/src/views/guidelines/getting_started.md

```diff
@@ -9,7 +9,7 @@ yarn add @opensearch-project/oui
 Note that OUI has [several `peerDependencies` requirements](package.json) that will also need to be installed if starting with a blank project. You can read more about other ways to [consume OUI][consuming].
 
 ```js
-yarn add @opensearch-project/oui @elastic/datemath moment prop-types
+yarn add @opensearch-project/oui @opensearch/datemath moment prop-types
```

Comment:
I don't think this is true: https://github.com/opensearch-project/oui/blob/main/package.json#L135

Reply:
That's right! Thanks Rocky for verifying. As OUI is still referencing @elastic/datemath at this point and have not updated it to @opensearch/datemath, I'll keep elastic references as it is in the code for now. cc @BSFishy

Reply:
Fixed in #204